runtime/src/lib.rs

```diff
@@ -237,18 +237,18 @@ impl pallet_social::Trait for Runtime {
 parameter_types! {
   pub const MinSpaceOwners: u16 = 1;
   pub const MaxSpaceOwners: u16 = u16::max_value();
```

Comment:
I just realized that `65536` space owners could be too much 😅 .
```suggestion
  pub const MaxSpaceOwners: u16 = 1000;
```